gallery/src/GalleryNav.tsx

```diff
@@ -125,7 +125,6 @@ function GalleryNav() {
       <NxFilterInput placeholder="Search RSC…"
                      value={filter}
                      onChange={setFilter}
-                     autoFocus
```

Comment:
Consensus opinion in the wild is that autofocus is bad for a11y.  @JenniKobylski has confirmed that we will stick to that opinion.